src/Files.App/Data/Items/ModifiableActionItem.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) 2024 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.App.Data.Items
+{
+	/// <summary>
+	/// Represents item for a hotkey that is registered for a <see cref="IRichCommand"/>.
+	/// </summary>
+	public class ModifiableActionItem : ObservableObject
```

Comment:
Should there be a property to set if customs args are supported?

Reply:
Added.